### DISCUSSION TITLE
cannot get a clean install of release i20230130 working on RPI3B  - LAN network fails

### DISCUSSION THREAD

**Discussion by TinkeringB** [rern/rAudio#1219] [Fixed Issues]
My SDcard got corrupted or something - realy not sure what happened; maybe a power-off from mains on a bad moment?
to the left my failing pi, and to the right a backup unit 
![afbeelding](https://user-images.githubusercontent.com/9662310/216427849-1cb21532-2ec4-4e89-8587-c5a847f23c49.png)

.... so I resolved to  a complete new flash and setup :-(
I downloaded i20230130 and put that on an sd card using win32imager. 
The pi boots, but after some time, it states the network is not connected (while all is wired up)

another image rAudio-1-64bit-20221117.img I still had in my download folder works right away, but cannot be updated anymore through the GUI and as that image had some other issues, that is a no-go either.

Sadly, I could not find an more recent package to install on here; only sourcecode and I am not too good with that..

**Comment by R101G** [comment]
Possibly another instance of the Pi4 netnames renaming bug?

Unless rern says otherwise, the easiest fix is to add _net.ifnames=0_ in the /boot/cmdline.txt file. Make sure everything in the file is on one line.

Edit: 

Just seen that you are on a Pi3b, so this might be that the new image allows for the renaming, but the renaming only happens on the Pi4. Is anyone in a position to check?

**Comment by TinkeringB** [comment]
Sadly no such luck; also after adding "net.ifnames=0" to the file, a fresh image results in a "network not connected" situation.
rAudio 20230130
kernel 5.15.89-3 
Raspberry Pi 3B rev1.2
SoC BCM2837 952MB

**Comment by hjfisch** [comment]
Have a look at "Ethernet interface stopped working" discussion some days earlier.I have Rpi3 too and this was working for me.

**Reply by TinkeringB** [reply]
Thank you; I read that, but how did you get access to the console; so how did you exit the chromium browser?

**Reply by hjfisch** [reply]
Sorry for answering so late.
I've a Linux System on my Laptop , that makes it easy for me to edit the sd card.

**Comment by R101G** [comment]
@bstrijker 

Check that a .network file in /etc/systemd/network matches what _ifconfig_ shows as the interface name. It might have been changed in the new image.

**Reply by TinkeringB** [reply]
Where and how to do that? 
Lacking IP access, I have no way to get into the console as far as I know to access that file and command.

**Reply by R101G** [reply]
@bstrijker  If you have no wifi either, edit the SD card. Edit the file /etc/systemd/network/en.network and change the line _Name = en*_ to _Name = et*_ and save the file as /etc/systemd/network/et.network. Reboot and see if that works.

**Comment by rern** [comment]
`rAudio-64bit-20230130.img.xz` - `/etc/systemd/network/en.network`:
```sh
[Match]
Name=en*

[Network]
DHCP=yes
DNSSEC=no

[Link]
RequiredForOnline=no
```

`rAudio-RPi2-20230130.img.xz` - `/etc/systemd/network/eth0.network`:
```sh
[Match]
Name=eth0

[Network]
DHCP=yes
DNSSEC=no

[Link]
RequiredForOnline=no
```

Note filename and `Name` of the 64bit which have been changed a couple versions ago.

**Reply by R101G** [reply]
@rern 

It looks as if the kernel only changes the Pi4 interface name, not the Pi3 or Pi2. If that is confirmed, would it be better to disable renaming in the boot file, and keep the eth*.network file?

**Comment by TinkeringB** [comment]
So I took the RPI2 image - the Pi3B now runs fine on that image.
Heavily overrated those 32 extra bits I guess.,

**Comment by ceejayemm** [comment]
I can also confirm this problem with a RPi 3B+ (previously have used 64 bit image with no problems) using lan/wired connection. Downloaded the i20230130 64 bit image to flash a new SD card.  After boot of RPi using the new card I could not connect to the RPi via SSH or via web app. Reflashed with (RPi 2) 32 bit version of the same date and had no problems starting  RPi. 

@rern - It does seem that the new 'en*' network setup for wired connections does not work with RPi 3 machines.

Chris

**Reply by R101G** [reply]
The easiest workaround is to edit the SD card after writing the image and add an /etc/systemd/network/et*.network file. The card then works in either model.

I would love to know the thinking behind this. Why is only the Pi4 changed?

**Comment by rern** [comment]
It's [Predictable Network Interface Names](https://www.freedesktop.org/wiki/Software/systemd/PredictableNetworkInterfaceNames/) which is not ready for RPi 3.

To fix the issue, `eth.network` must be restore:
```sh
sed 's/=en/=eth/' /etc/systemd/network/en.network > /etc/systemd/network/eth.network
```

New image files with the fix and a major kernel upgrade, 5.x.xx to 6.1.11, will be released soon.

**Comment by ceejayemm** [comment]
The new image (i20230212) seems to fix this problem on my RPi 3B+ which also includes the kernel update

Thanks @rern 

Chris
